### PR TITLE
Add some insights to the docs

### PR DESCRIPTION
**Pull request by Dworv**
This PR adds some tips from my experience using resetti to the docs. Hopefully, this can save some people some time.

### CODE REVIEW

File: doc/setup.md [hunk 1]
```diff
@@ -22,7 +22,8 @@ Minecraft (benchmarks show OpenJDK 20 to be slower), so get it if available.
 We will create a single instance, and then duplicate it as many times as
 needed. Start by creating an instance in MultiMC (or your fork) with the desired
 version (e.g. 1.16.1). Launch and close the game once; then, install Fabric by
-going to `Edit Instance` -> `Version`.
+going to `Edit Instance` -> `Version`. Unfortunatly, resetti does not currently, 
+work with Flatpak launchers.
```
Comment (woofdoggo) [inline doc/setup.md:26]: Typo: unfortunately

File: doc/setup.md [hunk 2]
```diff
@@ -62,6 +63,8 @@ mods you would like. Atum and Fast Reset are mandatory. The following are
   the below configuration to `.minecraft/config/sleepbg.json`.
   - You may have a better experience by tweaking some of these values, but they
     are a better starting point than the defaults.
+- If you want to make updating your settings easier, you can softlink your `config`
```
Comment (woofdoggo) [inline doc/setup.md:66]: Nitpick: This could be put below the line about SpeedrunIGT global options.

**Review by woofdoggo — CHANGES_REQUESTED** [review]
I think the disclaimer about Flatpak support is better placed with the other text about launchers, under the `Setting up Minecraft` header. Looks good otherwise.

File: doc/setup.md (hunk 1 above)
Comment (woofdoggo) [reply doc/setup.md:26]: Extra comma after currently

Comment (Dworv) [reply doc/setup.md:26]: Fixed

**Comment by Dworv** [comment]
I added the reording and put the warning into a cool Github block like this
image
I can make it normal if you don't like it.

File: doc/setup.md [hunk 3]
```diff
@@ -9,7 +9,11 @@ Click the icon in the upper left to view the table of contents.
 
 resetti has only been tested with [MultiMC](https://multimc.org) and forks such
 as [PrismLauncher](https://prismlauncher.org). No functionality is guaranteed
-when using the vanilla launcher or any other launcher.
+when using the vanilla launcher or any other launcher. 
+
+> **Warning**
+> resetti does not currently, work with launchers installed using Flatpak.
```
Comment (woofdoggo) [inline doc/setup.md:15]: Extra comma after currently

Comment (woofdoggo) [reply doc/setup.md:15]: After this fix it should be good to merge.

Comment (Dworv) [reply doc/setup.md:15]: Me and my commas smh

**Review by woofdoggo — APPROVED** [review]

**Pull request merged**